nbdev/templates/jekyll.tpl

```diff
@@ -7,6 +7,7 @@ sidebar: home_sidebar
 {% if resources.tags != "" and resources.tags != nil %}tags: {{resources.tags}}{% endif %}
 {% if resources.summary != "" and resources.summary != nil %}summary: "{{resources.summary}}"{% endif %}
 {% if resources.summary != "" and resources.summary != nil %}description: "{{resources.summary}}"{% endif %}
```

Comment:
@jph00 @sgugger  I noticed this line of code while I was doing this PR.  Looks like a bug, WDYT?  should the value should be description, not the summary again, right?